dns_connector/__init__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
The module should be named `connector_dns` rather than `dns_connector` to respect the convention to start with the main subject.